fastlane/lib/fastlane/actions/git_tag_exists.rb

```diff
@@ -0,0 +1,44 @@
+module Fastlane
+  module Actions
+    class GitTagExistsAction < Action
+      def self.run(params)
+        result = Actions.sh("git rev-parse -q --verify \"refs/tags/#{params[:tag]}\" || true", log: false).chomp
```

Comment:
Please shell escape your parameter instead of enclosing the argument in quotes:

```
git rev-parse -q --verify refs/tags/#{params[:tag].shellescape} || true
```